tests/integration/security/reachability_test.go

```diff
@@ -51,6 +51,9 @@ func TestReachability(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.reachability").
 		Run(func(t framework.TestContext) {
+			if t.Clusters().IsMulticluster() {
+				t.Skip("https://github.com/istio/istio/issues/39873")
```

Comment:
What exactly are the failures? If its the "connection reset" issues, I don't feel comfortable with this at all, because its (1) not related to these tests at all and (2) ~never going to be fixed. https://github.com/istio/istio/issues/38982 is the root cause.

I would much rather drop the test load (less parallel, less requests, etc). I would start with dropping from 15 requests to 3 before skipping (and drop the "reached all" if we really need to.